manageiq-operator/README.md

```diff
@@ -2,68 +2,158 @@
+    ```
+
+  3 - Push your new custom image to the registry:
+
+    ```bash
+    $ docker push docker.io/<your location>/manageiq-operator:latest
+    ```
+    
+  4 - Run your custom image from the registry:
+
+    ```bash
+    $ oc create -f deploy/operator.yaml
+    ```
+
++ #### Option 3: Run locally (on your local laptop/computer, outside of the cluster)
+
+  ```bash
+  $ operator-sdk run --local --namespace=<your namespace>
+  ```
+
+### Step 4. Run ManageIQ by creating the Custom Resource (CR)
 
 ```bash
-$ operator-sdk build docker.io/example/manageiq-operator:latest
+$ oc create -f deploy/crds/manageiq.org_v1alpha1_manageiq_cr.yaml
 ```
 
-2. Update the operator deployment with the new image:
+# Further Notes:
+
+## Customizing the installation
+
+ManageIQ can be run with an external Postgres or messaging server.  To do this, create the required OpenShift secret(s) with the correct parameters using the template(s) for [Postgres](templates/app/postgresql-secrets.yaml) or [messaging](/templates/app/kafka-secrets.yaml) and provide those secret names as `databaseSecret` and/or `kafkaSecret` in `manageiq.org_v1alpha1_manageiq_cr.yaml`.
+
+If you want to use a custom TLS certificate, one can be created with:
 
 ```bash
-$ sed -i 's|docker.io/manageiq/manageiq-operator:latest|docker.io/example/manageiq-operator:latest|g' deploy/operator.yaml
+oc create secret tls tls-secret --cert=tls.crt --key=tls.key` and setting the secret name as `tlsSecret` in `manageiq.org_v1alpha1_manageiq_cr.yaml`.
 ```
 
-3. Push the new image to the registry:
+## Creating an Operator Bundle
+
+Create the bundle image and push to an image registry
 
-```bash
-$ docker push docker.io/example/manageiq-operator:latest
 ```
+$ operator-sdk bundle create docker.io/example/manageiq-bundle:0.0.1 --image-builder podman --directory deploy/olm-catalog/manageiq-operator/0.0.1/ --channels alpha --default-channel alpha
+$ podman push docker.io/example/manageiq-bundle:0.0.1
+```
+## ManageIQ Simple Instance Example
 
-### Setup RBAC and deploy the operator
+Modify `deploy/crds/manageiq.org_v1alpha1_manageiq_cr.yaml` as follows:
 
-```bash 
-$ oc create -f deploy/role.yaml
-$ oc create -f deploy/role_binding.yaml
-$ oc create -f deploy/service_account.yaml
-$ oc create -f deploy/operator.yaml
+**Note:** The domain here will work for a Code Ready Containers cluster. Change it to one that will work for your environment.
+Additional parameters are available and documented in the Custom Resource Definition
+
+```yaml
+apiVersion: manageiq.org/v1alpha1
+kind: ManageIQ
+metadata:
+  name: miq
+spec:
+  applicationDomain: "miqproject.apps-crc.testing"
 ```
 
-### Create the CR to deploy ManageIQ
+## ManageIQ for OpenID-Connect Authentication Instance Example
+
+To run ManageIQ with OpenID-Connect Authentication, replace `Step 4.` from above with the following steps.
```

Comment:
This is a bit confusing,  the Replace step ....  4 (and 4c) are the same,  maybe make those say Step 5.  then add 4 for OpenID as (optional) with a and b there.

Reply:
OK. I'll try to improve this.